### ISSUE TITLE
[ERROR]

### ISSUE THREAD

**Issue by velourlawsuits** [victorchall/EveryDream2trainer#207]
Running locally on Windows 11 with an RTX 4080 16gb

Attached the stack trace and log files below:

[ModuleNotFoundError No module named 'triton.language'.txt](https://github.com/victorchall/EveryDream2trainer/files/11910264/ModuleNotFoundError.No.module.named.triton.language.txt)

[Mj1-20230629-161946.log](https://github.com/victorchall/EveryDream2trainer/files/11910266/Mj1-20230629-161946.log)

{
  "config": "train.json",
  "amp": false,
  "batch_size": 4,
  "ckpt_every_n_minutes": null,
  "clip_grad_norm": null,
  "clip_skip": 0,
  "cond_dropout": 0.04,
  "data_root": "D:\\Stable Diffusion\\EveryDream2trainer\\input\\Mj1",
  "disable_textenc_training": false,
  "disable_xformers": false,
  "flip_p": 0.0,
  "gpuid": 0,
  "gradient_checkpointing": true,
  "grad_accum": 1,
  "logdir": "logs",
  "log_step": 1,
  "lowvram": false,
  "lr": 3e-07,
  "lr_decay_steps": 0,
  "lr_scheduler": "constant",
  "lr_warmup_steps": 0,
  "max_epochs": 100,
  "notebook": false,
  "optimizer_config": "optimizer.json",
  "project_name": "Mj1",
  "resolution": 512,
  "resume_ckpt": "panopstor/EveryDream",
  "run_name": null,
  "sample_prompts": "sample_prompts.txt",
  "sample_steps": 5000000,
  "save_ckpt_dir": null,
  "save_ckpts_from_n_epochs": 0,
  "save_every_n_epochs": 25,
  "save_optimizer": false,
  "scale_lr": false,
  "seed": 555,
  "shuffle_tags": false,
  "validation_config": "validation_default.json",
  "wandb": false,
  "write_schedule": false,
  "rated_dataset": false,
  "rated_dataset_target_dropout_percent": 50,
  "zero_frequency_noise_ratio": 0.0,
  "save_full_precision": false,
  "disable_unet_training": false,
  "rated_dataset_target_dropout_rate": 50,
  "disable_amp": false,
  "useadam8bit": false,
  "no_prepend_last": false,
  "no_save_ckpt": false
}

**Comment by velourlawsuits** [comment]
Did a fresh install and fixed the issue.
